game.rb

```diff
@@ -12,18 +12,34 @@ def initialize game_view, io_handler, prompter, validator
 
   def play_game
     @io_handler.print(@prompter.prompt_for_game_greeting)
-    @io_handler.print(@prompter.prompt_for_word)
-    word = @validator.validate_selection(@prompter.prompt_for_word, 'is_a_word?')
-    word = word.upcase
+    word = player_one_play_game
     @game_view.clear_view
-    @io_handler.print(@prompter.prompt_for_letter)
     @io_handler.print(@game_view.blank_word(word))
-    letter = @validator.validate_selection(@prompter.prompt_for_letter, 'is_a_letter?')
-    letter = letter.downcase
-    @game_view.guessed_letters_view(letter)
-    @io_handler.print(@game_view.blank_word(@game_view.swap_letters_by_case(word, letter)))
-    @io_handler.print(@game_view.guessed_letters)
+    @io_handler.print(player_two_play_game(word))
+  end
+
+  def player_one_play_game
+    @io_handler.print(@prompter.prompt_for_word)
+    @validator.validate_selection(@prompter.prompt_for_word, 'is_a_word?').upcase
   end
-  
+
+  def player_two_play_game word
+    counter = 0
+    end_of_game = false
+    while ! end_of_game
+        @io_handler.print(@prompter.prompt_for_letter)
+        letter = @validator.validate_selection(@prompter.prompt_for_letter, 'is_a_letter?').downcase
+        @game_view.guessed_letters_view(letter)
+        word = @game_view.swap_letters_by_case(word, letter)
+        @io_handler.print(@game_view.blank_word(word))
+        @io_handler.print(@game_view.guessed_letters)
+        @io_handler.print(@prompter.prompt_for_word_guess)
+        word_guess = @io_handler.get_input
+        counter += 1
```

Comment:
We made a story for this but it would be nice if guessing a word counted as a guess!